client/src/boot/apollo/buildClient.js

```diff
@@ -10,7 +10,12 @@ import Config from 'lib/Config';
 const buildClient = async (baseUrl) => {
     const graphQLConfig = Config.getSection('SilverStripe\\Admin\\LeftAndMain').graphql;
     const cachedTypenames = graphQLConfig && graphQLConfig.cachedTypenames;
-    const fragmentData = await getGraphqlFragments(baseUrl, cachedTypenames);
+    let fragmentData;
+    try {
+      fragmentData = await getGraphqlFragments(baseUrl, cachedTypenames);
+    } catch (e) {
```

Comment:
Would it be practical to use a typed exception here? No biggy if it isn't.

If this behavior is only meant to be triggered during the behat test, it probably would not hurt to have comment explaining briefly why were are doing this. Otherwise someone will scratch their head on this one 2 years from now when looking at this.

Reply:
Yeah, could do. It's not just a behat fix, though. Anytime you're using admin without `asset-admin`, you'll get this error. Still a super edge case, but more reality-based than a testing anomaly. :)